build.gradle

```diff
@@ -286,6 +332,21 @@ dependencies {
 
     smokeTestCompile sourceSets.main.runtimeClasspath
     smokeTestCompile sourceSets.test.runtimeClasspath
+
+    contractTestCompile sourceSets.main.runtimeClasspath
+    contractTestCompile sourceSets.test.runtimeClasspath
+
+}
+
+
+
+pact {
+    publish {
+        pactDirectory = 'target/pacts'
+        pactBrokerUrl = System.getenv("PACT_BROKER_FULL_URL") ?: 'http://localhost:80'
```

Comment:
I actually copied this from another team and kept the same default port in case there was a reason for it.  In reality we are not currently running the broker locally.  We have one in the aat env but I envisage us normally publishing them as part of the pipeline (for which the hooks don't work :( ).